### PR TITLE
BM Added Creator Help link to creation tools

### PR DESCRIPTION
**Pull request by dmitrii-zolotukhin**
- [x] Add link to sparklever.se/ity for /admin

closes sparkletown/internal-sparkle-issues#1141

### CODE REVIEW

File: src/pages/Admin/Admin.tsx [hunk 1]
```diff
@@ -125,6 +125,9 @@ const VenueList: React.FC<VenueListProps> = ({
         <Link to="/admin/venue/creation" className="btn btn-primary">
           Create a venue
         </Link>
+        <a href={SPARKLEVERSE_ITY} target="_blank" rel="noreferrer">
+          SparkleVersity Tutorials & Creator Guides
+        </a>
```
Comment (goran-peoski-work) [inline src/pages/Admin/Admin.tsx:130]:
```suggestion
        <a href={SPARKLEVERSE_ITY} {...getExtraLinkProps(true)}>
          SparkleVersity Tutorials & Creator Guides
        </a>
```

File: src/settings.ts [hunk 2]
```diff
@@ -28,6 +28,7 @@ export const SPARKLEVERSE_TERMS_AND_CONDITIONS_URL =
   "https://sparklever.se/terms-of-use/";
 export const SPARKLEVERSE_PRIVACY_POLICY =
   "https://sparklever.se/privacy-policy/";
+export const SPARKLEVERSE_ITY = "https://sparklever.se/ity/";
```
Comment (goran-peoski-work) [inline src/settings.ts:31]: Maybe rename to SPARKLEVERSEITY_URL or anything that ends in `_URL`

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Minor code style changes and good to go

File: src/pages/Admin/Admin.tsx (hunk 1 above)
Comment (dmitrii-zolotukhin) [reply src/pages/Admin/Admin.tsx]: @goran-peoski-work fixed in e4059b28059129a2c241deb31894acd1d0feac1a

File: src/settings.ts (hunk 2 above)
Comment (dmitrii-zolotukhin) [reply src/settings.ts]: @goran-peoski-work fixed in e4059b28059129a2c241deb31894acd1d0feac1a

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Pull request merged**